CHANGELOG.rst

```diff
@@ -1,3 +1,59 @@
 =========
 Changelog
 =========
+
+
+Unreleased
+==========
+
+Added
```

Comment:
Please add an entry here describing the new visibility simulator class!

Reply:
Under the "Unreleased" category or v0.1.0 [2019.08.28] or a new version?

Reply:
I would put it under v0.1.1 [yyyy.mm.dd], following the format of the previous entry, and giving an overview of what has been added/changed/removed/etc. Also remember to update the version in the code (it lives in the VERSION file).

Reply:
err, sorry, v0.2.0? I think we're doing semantic versioning, with major.minor.patch, so this is a new minor version, since it adds new features but doesn't break the API

Reply:
yup, 0.2.0

Reply:
Done. Take a look.